docs/next.config.js

```diff
@@ -33,10 +33,8 @@ module.exports = {
   },
   typescript: {
     // Motivated by https://github.com/vercel/next.js/issues/7687
-    ignoreDevErrors: true,
```

Comment:
Removed in https://github.com/vercel/next.js/pull/12941 (I hate this kind of PRs, when you look at it, you have no idea why `ignoreDevErrors` is gone. And of course the PR is locked, so you can't ask 🙈)